op-node/rollup/derive/engine_controller.go

```diff
@@ -327,7 +327,9 @@ func (e *EngineController) InsertUnsafePayload(ctx context.Context, envelope *et
 	// Check if there is a finalized head once when doing EL sync. If so, transition to CL sync
 	if e.syncStatus == syncStatusWillStartEL {
 		b, err := e.engine.L2BlockRefByLabel(ctx, eth.Finalized)
-		isTransitionBlock := e.rollupCfg.Genesis.L2.Number != 0 && b.Hash == e.rollupCfg.Genesis.L2.Hash
+		// [Kroma: ZKT to MPT]
+		isTransitionBlock := e.rollupCfg.Genesis.L2.Number != 0 && b.Hash == e.rollupCfg.Genesis.L2.Hash || e.rollupCfg.IsKromaMPT(ref.Time)
```

Comment:
I wonder why starting EL sync if the unsafe payload is after Kroma MPT time

Reply:
I checked again, it seems we don’t need to make any changes to this part.